src/runtime/RegExpObject.h

```diff
@@ -179,6 +180,7 @@ class RegExpObject : public Object {
 
     SmallValue m_lastIndex;
     const String* m_lastExecutedString;
+    bool m_isRegExpPrototypeObject;
```

Comment:
How about make class for this?
I think derive RegexpObject and define  
```
virtual bool isRegExpPrototypeObject() const
{
    return true;
}
```
is better than add new member variable for a each RegExpObject.
because it can reduce memory usage/.. (some js codes generate a lots of RegExpObject)